resources/lib/sutils.py

```diff
@@ -54,17 +53,17 @@ def service(self):
         self.sleep(sleep_time)
 
         try:
-            self.last_run = float(self.cache.get("subscription.last_run")) #time.time()
+            self.last_run = float(self.cache.get("subscription.last_run"))
         except:
             self.last_run = time.time()
             self.cache.set("subscription.last_run", str(self.last_run))
             pass
 
-        if not xbmc.abortRequested and time.time() > self.last_run + 24 * 3600:
+        if not xbmc.abortRequested and time.time() > self.last_run:
             self.evalSchedules()
 
         while not xbmc.abortRequested:
-            if(time.time() > self.last_run + 24 * 3600):
+            if(time.time() > self.last_run + 600): #evaluate subsciptions every 10 minutes
```

Comment:
According to [PEP 8](https://www.python.org/dev/peps/pep-0008/#inline-comments), this is a malformed comment.